docs/CMakeLists.txt

```diff
@@ -31,7 +31,9 @@ set(DOXYGEN_GENERATE_TAGFILE "${CMAKE_CURRENT_BINARY_DIR}/html/${CMAKE_PROJECT_N
 set(DOXYGEN_TAGFILES "${KOKKOS_TAG_FILE}")
 
 #---- Extract files for documentation.
-get_target_property(KokkosUtils_TARGET_FILES_FOR_DOC KokkosUtils INTERFACE_SOURCES)
+get_target_property(KokkosUtils_TARGET_FILES_FOR_DOC kokkosutils INTERFACE_HEADER_SETS)
+get_target_property(KokkosUtils_TARGET_FILES_FOR_DOC kokkosutils HEADER_SET_${KokkosUtils_TARGET_FILES_FOR_DOC})
```

Comment:
Should we use two different variables names?

Reply:
Not needed, I did it on purpose.